exchanges/deribit/deribit_websocket.go

```diff
@@ -494,15 +494,15 @@ func (d *Deribit) processTrades(respRaw []byte, channels []string) error {
 	if len(tradeList) == 0 {
 		return fmt.Errorf("%v, empty list of trades found", common.ErrNoResponse)
 	}
-	tradeDatas := make([]trade.Data, len(tradeList))
-	for x := range tradeDatas {
+	tradesData := make([]trade.Data, len(tradeList))
+	for x := range tradesData {
```

Comment:
not a change request, range is a bit better if you are ranging over the item you are utilising/has data in that range because if somebody in future changed the `x := range` to a `x, d := range` then it can get buggy.